README.md

```diff
@@ -36,7 +36,7 @@
       API Docs
     </a>
     <span> | </span>
-    <a href="https://github.com/rustasync/tide/blob/master/.github/CONTRIBUTING.md">
+    <a href="./.github/CONTRIBUTING.md">
```

Comment:
The URLs were intentionally long-form so they correctly link from e.g. crates.io

Reply:
Oh. Oops.